core/src/test/java/org/infinispan/statetransfer/RemoteGetDuringStateTransferTest.java

```diff
@@ -203,6 +203,8 @@ public void testScenario_111() throws Exception {
     * ISPN-3315: the remote get is trigger in stable state and the reply received after the rebalance_start command.
     * However, the old owner will receive the request after the state transfer and he no longer has the key.
     */
+   // TODO: does it make sense to fix this test by waiting for the OOTE and then allowing the topology?
+   @Test(enabled = false, description = "Remote get is blocked due to origin topology < old owner topology")
```

Comment:
Originator has topology 4 and sends a remoteGet - old owner has already installed topology 5, so it throws OOTE which is propagated to originator. When it sees OOTE, it waits for next topology (5) but that never comes because the test blocks topology updates on originator. Therefore, the test fails with timeout.